resources/components/base/Suggestion.vue

```diff
@@ -0,0 +1,147 @@
+<template>
+	<div
+		class="mw-suggestion"
+		tabindex="0"
+		v-bind:class="classObject"
+		v-on:click="onClick"
+		v-on:keydown.enter="onClick"
+	>
+		<label class="mw-suggestion__label">
+			<slot />
```

Comment:
See above comment about whether we need to use `<slot>` here. This could also just be `{{ text }}`